src/js/components/Carousel/index.d.ts

```diff
@@ -3,19 +3,24 @@ import {
   A11yTitleType,
   AlignSelfType,
   GridAreaType,
+  HeightType,
   MarginType,
+  WidthType,
 } from '../../utils';
 
 export interface CarouselProps {
   a11yTitle?: A11yTitleType;
   activeChild?: number;
   alignSelf?: AlignSelfType;
+  continuous?: boolean;
```

Comment:
Can we rename this property to be "wrap"? I think that conveys the behavior a bit better than "continuous". I think "continuous" feels more like it's associated with "play", which isn't as true here.